lib/ja_serializer/dsl.ex

```diff
@@ -72,30 +72,43 @@ defmodule JaSerializer.DSL do
   end
 
   @doc false
-  defmacro __before_compile__(_env) do
+  defmacro __before_compile__(env) do
     quote do
       def __relations,  do: @relations
       def __location,   do: @location
       def __attributes, do: @attributes
+
+      unquote(define_inlined_attributes_map(env))
+    end
+  end
+
+  defp define_inlined_attributes_map(env) do
+    attributes = Module.get_attribute(env.module, :attributes)
+    conn = quote do: conn
+    struct = quote do: struct
```

Comment:
Just to explain this a bit.

Here I "quoting" simple variable. That way I can use this variable signature below in the map AST, and link it later in the `def inlined_attributes_map` to the function argument.

See macro in Plug.Builder here https://github.com/elixir-lang/plug/blob/v1.3.5/lib/plug/builder.ex#L130 where I saw this pattern.